lib/rubocop/cop/style/single_line_do_end.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for single-line `do`...`end` blocks.
```

Comment:
You might also add a note about why the auto-correct doesn't replace `do/end` with `{}`, as perhaps this won't be clear for some people.

Reply:
Indeed, I've added an explain for that.